plugins/labhub.py

```diff
@@ -143,7 +137,6 @@ def callback_message(self, msg):
                 # send the invite
                 self.send(msg.frm,
                           self.INVITE_SUCCESS['newcomers'].format(user))
-                self.TEAMS[self.GH_ORG_NAME + ' newcomers'].invite(user)
                 self.invited_users.add(user)
```

Comment:
Remove this line, also change regex such that it will not respond to `invite me`

Reply:
1. Would remove the regex 👍 
2. Would change `self.invited_users` to `self.helloed_users`, since it is just tracking if it needs to trigger again or not

Reply:
Also, when maintainer invites, invite message should be different right? as we changed the message